src/change/sdp.rs

```diff
@@ -925,6 +926,19 @@ fn update_session(session: &mut Session, sdp: &Sdp) {
     if has_transport_cc && has_twcc_header {
         session.enable_twcc_feedback();
     }
+
+    let has_ssrc_signalling = sdp.media_lines.iter().any(|m| {
+        m.attrs
+            .iter()
+            .any(|a| matches!(a, MediaAttribute::SsrcGroup { .. }))
+    });
+
+    if has_ssrc_signalling {
+        // If the remote indicated explicit signalling of simulcast via ssrc-group we disabled rid
```

Comment:
Maybe we should log something here

Reply:
Maybe a comment about why we are doing it? A link to the FF issue?